sql/updates/latest-dev.sql

```diff
@@ -94,3 +94,33 @@ GRANT SELECT ON _timescaledb_catalog.chunk_id_seq TO PUBLIC;
 GRANT SELECT ON _timescaledb_catalog.chunk TO PUBLIC;
 
 -- end recreate _timescaledb_catalog.chunk table --
+
+-- First we update the permissions of the compressed hypertables to
+-- match the associated hypertable.
+WITH
+    hypertables AS (
+	SELECT format('%I.%I', ht.schema_name, ht.table_name)::regclass AS hypertable_oid,
+	       format('%I.%I', ct.schema_name, ct.table_name)::regclass AS compressed_hypertable_oid
+	FROM _timescaledb_catalog.hypertable ht
+	JOIN _timescaledb_catalog.hypertable ct ON ht.compressed_hypertable_id = ct.id
+    )
+UPDATE pg_class
```

Comment:
Do we need an `UPDATE ONLY` here or do we not need to worry about inheritance?

Reply:
Not for `pg_class`, no.